$NetBSD: patch-tests_CMakeLists.txt,v 1.1 2020/09/13 18:45:36 otis Exp $

Search for sshd also in /usr/lib/ssh where it is located
in SunOS/SmartOS.

--- tests/CMakeLists.txt.orig	2020-09-10 12:43:04.000000000 +0000
+++ tests/CMakeLists.txt
@@ -117,7 +117,8 @@ find_program(SSHD_EXECUTABLE
              PATHS
                 /sbin
                 /usr/sbin
-                /usr/local/sbin)
+                /usr/local/sbin
+                /usr/lib/ssh)
 
 if (CLIENT_TESTING OR SERVER_TESTING)
     find_package(socket_wrapper 1.1.5 REQUIRED)
